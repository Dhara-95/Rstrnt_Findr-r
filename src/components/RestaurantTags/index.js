import React from "react";

const RestaurantTags = (props) => {
  const { tags } = props;

  return (
    <div>
      {tags.map((tag, index) => {
        return (
          <span className="text-grey" key={index}>
            {(index ? ", " : "") + tag.title}{" "}
          </span>
        );
      })}
    </div>
  );
};

export { RestaurantTags };
