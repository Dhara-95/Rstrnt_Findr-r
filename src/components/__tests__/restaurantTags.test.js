import React from "react";
import renderer from "react-test-renderer";

import { RestaurantTags } from "../RestaurantTags";

it("displays the Restaurant tags correctly", () => {
  const tree = renderer
    .create(<RestaurantTags tags={[{ title: "Cafe" }]} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});

it("displays the Restaurant tags correctly when multiple tags are passed in", () => {
  const tree = renderer
    .create(
      <RestaurantTags
        tags={[{ title: "Cafe" }, { title: "Caribbean" }, { title: "Kebab" }]}
      />
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
