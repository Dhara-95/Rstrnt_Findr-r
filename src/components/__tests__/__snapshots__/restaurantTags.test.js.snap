// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`displays the Restaurant tags correctly 1`] = `
<div>
  <span
    className="text-grey"
  >
    Cafe
     
  </span>
</div>
`;
